### PR TITLE
Distributed training on CPU

### PR DESCRIPTION
**Pull request by Ilya-Krylov**

### CODE REVIEW

File: tools/train.py [hunk 1]
```diff
@@ -183,7 +193,11 @@ def main():
         distributed = False
     else:
         distributed = True
-        init_dist(args.launcher, **cfg.dist_params)
+        if torch.cuda.is_available():
+            init_dist(args.launcher, **cfg.dist_params)
+        else:
+            cfg.dist_params['backend'] = 'gloo'
```
Comment (druzhkov-paul) [inline tools/train.py:199]: Does this introduce an extra dependency or anything that's worth mentioning in the docs?

Comment (Ilya-Krylov) [reply tools/train.py:199]: By default for Linux, the Gloo and NCCL backends are built and included in PyTorch distributed (NCCL only when building with CUDA).

**Review by druzhkov-paul — APPROVED** [review]

**Comment by Ilya-Krylov** [comment]
run tests

**Comment by Ilya-Krylov** [comment]
run tests

**Pull request merged**